clang/include/clang/AST/Expr.h

```diff
@@ -3182,11 +3182,12 @@ class CallExpr : public Expr {
 
   /// Returns the WarnUnusedResultAttr that is either declared on the called
   /// function, or its return type declaration.
```

Comment:
The comment should be expanded to also explain what the `NamedDecl*` in the pair is supposed to be